scripts/snippet_mover_report.txt

```diff
@@ -0,0 +1,78 @@
+Total snippet module count: 91
+Total snippets found: 934
+Moved snippet module count: 91
+Shared snippet count: 45
+
+
+
+
+******************************************************************************
+
+
+
+12 snippets not found in tests:
+tests/integration/docusaurus/connecting_to_your_data/database/athena_python_example.py Datasource dict config
+great_expectations/checkpoint/actions.py empty dict
+great_expectations/checkpoint/actions.py get_docs_sites_urls
+great_expectations/checkpoint/actions.py iterate
+full config for pandas runtime Data Source with blank data asset identifiers
+full pandas runtime Data Source configuration
+full configuration for sql runtime Data Source
+full configuration for sql configured Data Source
+full configuration for sql inferred Data Source
+spark Data Source configuration up to adding an empty runtime data connector
+spark Data Source configuration up to adding an empty inferred data connector
+spark Data Source configuration up to adding an empty configured data connector
+
+
+
+******************************************************************************
+
+
+
+0 Unmoved snippet modules:
+
+
+
+
+******************************************************************************
+
+
+
+36 modules contain snippets but are not referenced by the docs:
```

Comment:
This document should be deleted soon, but is currently useful for followup work, since these files can possibly be deleted.